SlevomatCodingStandard/Sniffs/Classes/AbstractPropertyAndConstantSpacingSniff.php

```diff
@@ -0,0 +1,102 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+use function assert;
+use function in_array;
+use const T_COMMENT;
+use const T_CONST;
+use const T_DOC_COMMENT_OPEN_TAG;
+use const T_FUNCTION;
+use const T_PRIVATE;
+use const T_PROTECTED;
+use const T_PUBLIC;
+use const T_SEMICOLON;
+use const T_STRING;
+use const T_VAR;
+use const T_VARIABLE;
+
+abstract class AbstractPropertyAndConstantSpacingSniff implements Sniff
+{
+
+	/** @var int */
+	public $minLinesCountBeforeWithComment = 1;
```

Comment:
It think it would be better check exactly the count of lines. Why is `min` here?

Reply:
This provides you with more flexibility. If you want to force exact number of lines, just set both min and max to the same number. Let's say someone likes to stack the properties, so they'll set min to 0. But they would also like to add an empty line in between properties to visually separate let's say, two groups. To do that, they set max to 1. You wouldn't be able to do that with just one value. It would be either stacking or space.

Reply:
Ok.